client/my-sites/comments/main.jsx

```diff
@@ -10,21 +10,33 @@ import { localize } from 'i18n-calypso';
 import Main from 'components/main';
 import PageViewTracker from 'lib/analytics/page-view-tracker';
 import DocumentHead from 'components/data/document-head';
+import CommentList from './comment-list';
+import CommentNavigation from './comment-navigation';
 
 export class CommentsManagement extends Component {
 
 	static propTypes = {
 		basePath: PropTypes.string,
+		siteSlug: PropTypes.string,
+		status: PropTypes.string,
 		translate: PropTypes.func,
 	};
 
 	render() {
-		const { translate, basePath } = this.props;
+		const {
+			basePath,
+			siteSlug,
+			status,
+			translate,
+		} = this.props;
 		return (
-			<Main className="comments">
+			<Main className="comments" wideLayout>
 				<PageViewTracker path={ basePath } title="Manage Comments" />
 				<DocumentHead title={ translate( 'Manage Comments' ) } />
-				<div>Hello World!</div>
+				<div className="comments__primary">
+					<CommentNavigation siteSlug={ siteSlug } status={ status } />
+					<CommentList />
```

Comment:
Nothing in particular, but it does feel like we're reinventing the wheel a bit. I remember evaluating a few more InfiniteList implementations while doing a performance pass, but having to end up deferring to our current component. I remember liking https://github.com/orgsync/react-list

If we feel that our current InfiniteList is not up to snuff, I would recommend adding a task to  https://github.com/Automattic/wp-calypso/projects/32 along with some reasoning in the issue of why we need another implementation.